packages/core/src/chain/index.ts

```diff
@@ -0,0 +1,148 @@
+import type { Chain } from "viem";
+import {
+  arbitrum as vab,
+  arbitrumGoerli as vabg,
+  arbitrumSepolia as vabs,
+  base as vbase,
+  baseGoerli as vbaseg,
+  baseSepolia as vbases,
+  goerli as vgo,
+  mainnet as vmain,
+  optimism as vop,
+  optimismGoerli as vopg,
+  optimismSepolia as vops,
+  polygon as vpg,
+  polygonMumbai as vpgm,
+  sepolia as vsep,
+} from "viem/chains";
+
+export const arbitrum: Chain = {
+  ...vab,
+  rpcUrls: {
+    ...vab.rpcUrls,
+    alchemy: {
+      http: ["https://arb-mainnet.g.alchemy.com/v2"],
+    },
+  },
+};
```

Comment:
I'm not sure I understand the need for this. Is this the final state or are we planning on updating in V3?

I think in `core` we should just use the raw `viem/chains` definitions, and then we can add our own chain overrides in `aa-alchemy` for easy use with our APIs.

Reply:
I think want it in aa-core so any dependent package (aa-alchemy, aa-signers, aa-ethers, aa-accounts) gets it for free. Also, a dev may benefit from using this even if they're just using aa-core.

Reply:
How does `aa-core` get the RPC url to use for requests? Does it always use `chain.alchemy + API key`? That logic doesn't seem right to me to be in core.

Reply:
To be more specific about why I don't like this, it requires a new version (and devs to upgrade) each time we support a new chain, else it doesn't look like we support the chain at all. When in reality all that is needed is to grab the chain definition from viem (or create a custom chain) and drop your alchemy RPC URL into it. We should be training users to do this instead of relying on our definitions.

If all we are adding to the chain definition is an RPC URL I don't see the need to redefine these, users can pass in chain URLs just as easily as they can pass in an API key.

Reply:
Make sense and I imagine that's a reason viem moved away from this. Fwiw, we do this in alchemy-sdk i.e. updating versions to add chains. 

I think the idea is we view chain support like a new feature, thus prompted a new minor version semantically. And if we're out of date on chains supported, 1. we shouldn't be re: respecting a rollout, and 2. it feels fine if we view it as a feature to add in an upcoming version

I do like the convenience of having the rpc urls ready to use out-of-the-box, and it gives us a marker of supported chains by Alchemy. And if a dev wants to use their own alchemy rpc url (if they're out of date), they still can in aa-core.

Reply:
The problem is less so us respecting the rollout, its more-so ensuring the developers upgrade their SDKs before complaining that we don't support a chain.

The marker of supported chains by alchemy at the end of the day should be our dashboard and the ability to create an app (i.e. get an RPC url) and populate a viem chain config.

This will be much more sustainable in the N-chains future

Reply:
> we do this in alchemy-sdk i.e. updating versions to add chains.

Understood, and this is something I'm specifically trying to move us away from.

Reply:
what I can export from aa-alchemy is a function called `defineAlchemyChain` which takes in a viem `chain` + `rpcUrl` and spits out a chain with the right config